src/main/java/dinkplugin/notifiers/GambleNotifier.java

```diff
@@ -0,0 +1,138 @@
+package dinkplugin.notifiers;
+
+import com.google.common.collect.ImmutableList;
+import dinkplugin.message.NotificationBody;
+import dinkplugin.message.NotificationType;
+import dinkplugin.notifiers.data.GambleNotificationData;
+import dinkplugin.notifiers.data.SerializedItemStack;
+import dinkplugin.util.ItemSearcher;
+import dinkplugin.util.ItemUtils;
+import dinkplugin.util.Utils;
+import lombok.NonNull;
+import lombok.Value;
+import lombok.extern.slf4j.Slf4j;
+import net.runelite.client.game.ItemManager;
+import org.apache.commons.lang3.StringUtils;
+import org.apache.commons.lang3.tuple.Pair;
+import org.jetbrains.annotations.Nullable;
+import org.jetbrains.annotations.VisibleForTesting;
+
+import javax.inject.Inject;
+import javax.inject.Singleton;
+import java.util.List;
+import java.util.regex.Matcher;
+import java.util.regex.Pattern;
+import java.util.stream.Collectors;
+import java.util.stream.Stream;
+
+@Slf4j
+@Singleton
+public class GambleNotifier extends BaseNotifier {
+    // itemName (x quantity)![ tertiaryReward!] gc: n
+    private static final Pattern GAMBLE_REGEX = Pattern.compile(
+        "^(.+?)!\\s*(.+!)?\\s+High level gamble count: (\\d+).*"
+    );
+    private static final Pattern ITEM_QUANTITY_REGEX = Pattern.compile("^(.+?)\\(x (\\d+)\\)$");
+    private static final List<String> RARE_LOOT = ImmutableList.of("dragon chainbody", "dragon med helm");
```

Comment:
Let's make this a `Collection<String>` since it's really describing a set... but I'm fine with keeping the concrete implementation as list given the likely faster contains check in practice due to low n (despite theoretically worse algo complexity)

Reply:
Now that you bring it up, it doesn't really feel right as a list